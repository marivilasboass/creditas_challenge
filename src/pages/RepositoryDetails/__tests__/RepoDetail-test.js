import 'react-native';
import React from 'react';
import RepoDetail from '~/pages/RepositoryDetails';
import renderer from 'react-test-renderer';

const navigation = {
  getParam: () => ({id: 1}),
};

it('renders RepositoryList correctly (snapshot)', () => {
  const component = renderer
    .create(<RepoDetail navigation={navigation} />)
    .toJSON();
  expect(component).toMatchSnapshot();
});
