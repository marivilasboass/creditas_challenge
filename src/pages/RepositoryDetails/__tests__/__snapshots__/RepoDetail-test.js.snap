// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders RepoList correctly (snapshot) 1`] = `
<ActivityIndicator
  animating={true}
  color="#999999"
  hidesWhenStopped={true}
  size="small"
  style={
    Object {
      "flex": 1,
    }
  }
/>
`;
